src/builder.rs

```diff
@@ -189,11 +191,37 @@ impl<'req, 'dev, 'win, W: HasRawWindowHandle> PixelsBuilder<'req, 'dev, 'win, W>
         self
     }
 
-    /// Set the default clear color.
+    /// Set the blend state.
+    ///
+    /// Allows customization of how to mix the new and new existing pixels in a texture
```

Comment:
Looks like a small typo here.

```suggestion
    /// Allows customization of how to mix the new and existing pixels in a texture
```